Add native v4 workflow attribute events (#2226)
vercel/workflow: docs/content/docs/v5/observability/attributes.mdx

[`experimental_setAttributes`](/docs/api-reference/workflow/experimental-set-attributes) attaches plaintext string metadata to the current workflow run. These attributes are displayed in observability CLI/UI.
In the future, you'll be able to search and filter runs by attributes.

> You can also seed attributes without a follow-up write when starting a run:

Author: ```suggestion
You can also seed any attributes directly when starting a run:
```

Author: Applied in `3dfd8be2f`.
